src/Infrastructure/Connector/Action/BatchDeleteProductVisiblity.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ergonode\ExporterShopware6\Infrastructure\Connector\Action;
+
+use Ergonode\ExporterShopware6\Infrastructure\Connector\AbstractAction;
+use Ergonode\ExporterShopware6\Infrastructure\Model\BatchVisibilitiesProduct;
+use GuzzleHttp\Psr7\Request;
+use Symfony\Component\HttpFoundation\Request as HttpRequest;
+
+class BatchDeleteProductVisiblity extends AbstractAction
+{
+    private const URI = '/api/_action/sync';
+
+    protected BatchVisibilitiesProduct $batchVisibilitiesProduct;
+
+    public function __construct(BatchVisibilitiesProduct $batchVisibilitiesProduct)
+    {
+        $this->batchVisibilitiesProduct = $batchVisibilitiesProduct;
+    }
+
+    public function getRequest(): Request
+    {
+        return new Request(
+            HttpRequest::METHOD_POST,
+            $this->getUri(),
+            $this->buildHeaders(),
+            $this->buildBody(),
+        );
+    }
+
+    /**
+     * @param string|null $content
+     *
+     * @return null
+     */
+    public function parseContent(?string $content)
```

Comment:
return type can be void

Reply:
Inne implementacje tej funkcji w ramach interfejsu mają w komentarzu, że dana funkcja musi zwracać `array|object|string|null`. Nie pamiętam w tym momencie dlaczego ale przy testach funkcjonalnych próba użycia `return ;` powodowała błedy